README.md

```diff
@@ -218,6 +221,46 @@ Options:
 - `bitLength`: Number of id bits. (default: 128)
 - `multibase`: Account for multibase encoding. (default: true)
 
+### `generateKeySeed(options)`
+
+`generateKeySeed()` and `decodeKeySeed()` methods are for creating  and
+decoding key pair seeds which will be used for auto-generating app ids.
+
+`generateKeySeed()` generates a string key seed ID.
```

Comment:
A seed is not an identifier, it's used to generate one.

```suggestion
`generateKeySeed()` generates a secret key seed encoded as a string that
can be stored and later used to generate a key pair. The public key from
the key pair can be used as an identifier.
```

Reply:
Updated bcc742c